src/main/java/pages/ProductPage.java

```diff
@@ -1,10 +1,27 @@
 package pages;
 
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
+import lombok.Getter;
+
+@Getter
 public class ProductPage extends PageObject {
 
-    ProductPage(final WebDriver driver) {
+    @FindBy(xpath = "//input[@id='add-to-cart-button-19']")
+    private WebElement addToCartButton;
+
+    @FindBy(xpath = "//input[@id='add-to-wishlist-button-19']")
+    private WebElement addToWishButton;
+
+    @FindBy(xpath = "//span[@class='cart-label']")
+    private WebElement ShoppingCartLink;
+
+    @FindBy(xpath = "//span[@class='wishlist-label']")
+    private WebElement WishListLink;
+
+    public ProductPage(final WebDriver driver) {
         super(driver);
     }
```

Comment:
Why web elements wrapped in the constructor?

Reply:
they are not wrapped in constructor